_data/projects/Hoodie.yml

```diff
@@ -0,0 +1,11 @@
+name: Hoodie
+desc: Generic Backend for frontend Apps
+site: https://github.com/dahlbyk/up-for-grabs.net
+tags:
+- javascript
+- web
+- offlinefirst
+- nobackend
+upforgrabs:
+  name: starter
+  link: http://go.hood.ie/hoodie-starter-issues
```

Comment:
How about this URL: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aopen+is%3Aissue+org%3Ahoodiehq+label%3Astarter+label%3Aready+ – does that work with your labels showing the number of open issues?